src/pages/companies-list/companies-list.js

```diff
@@ -0,0 +1,23 @@
+import React, { useState, useEffect } from 'react';
+import CompaniesListing from '../../components/template/companies-listing-grid';
+import instance from '../../service/api';
+import Template from '../../components/template/template.component';
+import { useSelector } from 'react-redux';
+
+const CompaniesList = () => {
+  // eslint-disable-next-line
```

Comment:
why do we need this?

Reply:
Because if we don't disable eslint it will point const user from useSelector as initialized but never used, and the project will be compiled with warnings